backend/scripts/sync_prod_to_sqlite.py

```
@@ -0,0 +1,284 @@
+        'aws', 'ssm', 'get-parameter', '--name', PROD_PARAM,
+        '--with-decryption', '--query', 'Parameter.Value', '--output', 'text',
+    ])
+    rest = url.split('://', 1)[1]
+    creds, hostpart = rest.split('@', 1)
+    user, password = creds.split(':', 1)
+    hostport, dbname = hostpart.split('/', 1)
+    host, _, port = hostport.partition(':')
+    port = port or '5432'
+
+    BACKUP_DIR.mkdir(exist_ok=True)
+    out = BACKUP_DIR / f'tally_prod_{datetime.now():%Y%m%d_%H%M%S}.sql'
```

Ruff (0.16.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)